webvtt.go

```diff
@@ -203,6 +206,50 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	return
 }
 
+func extractVoiceNameAndText(line string) (string, string) {
```

Comment:
Can you rename the method to `parseTextWebVTT(i string) (o Line)` ?

Reply:
done